website/client/js/controllers/tavernCtrl.js

```diff
@@ -2,7 +2,7 @@
 
 habitrpg.controller("TavernCtrl", ['$scope', 'Groups', 'User', 'Challenges',
     function($scope, Groups, User, Challenges) {
-      Groups.tavern()
+      Groups.tavern(true)
```

Comment:
We shouldn't force a lookup of the Tavern every time, that'd be a _lot_ of requests. 

I think the source of the chat bug is that the cached version of the Tavern isn't persisting the new messages that get added.